### PR TITLE
adapt CI tests to use compiled_rmsnorm

### PR DESCRIPTION
**Pull request by tianyu-l**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #451
* #443

### CODE REVIEW

File: test_runner.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def build_test_list():
                     "--experimental.pipeline_parallel_split_points layers.4",
                     "--experimental.pipeline_parallel_schedule 1f1b",
                     "--training.data_parallel_degree 1",
-                    "--model.norm_type rmsnorm",  # fused_rmsnorm crashes with PP
+                    "--model.norm_type rmsnorm",  # compiled_rmsnorm / fused_rmsnorm crashes with PP
```
Comment (awgu) [inline test_runner.py:72]: curious what is the reason that compiled RMSNorm is incompatible with PP

File: test_runner.py [hunk 2]
```diff
@@ -257,7 +253,7 @@ def build_test_list():
                     "--experimental.pipeline_parallel_degree 4",
                     "--experimental.pipeline_parallel_split_points layers.1,layers.2,layers.3,layers.4,layers.5,layers.6,layers.7",
                     "--experimental.pipeline_parallel_schedule interleaved_1f1b",
-                    "--model.norm_type rmsnorm",  # fused_rmsnorm throws cuda context error with pp
+                    "--model.norm_type rmsnorm",  # compiled_rmsnorm / fused_rmsnorm throws cuda context error with pp
```
Comment (awgu) [inline test_runner.py]: e.g. is this cuda context error the same reason as above for "crashes with PP"?

Comment (tianyu-l) [reply test_runner.py]: Yes. It seems nobody understands why the error happens at this moment.

Comment (awgu) [reply test_runner.py]: If it is the same error, maybe we should make the commit uniform?

**Comment by tianyu-l** [comment]
see updates in https://github.com/pytorch/torchtitan/pull/535

**Pull request closed without merging**